dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/TrackedEntityInstanceController.java

```diff
@@ -390,10 +346,11 @@ public void queryTrackedEntityInstancesCsv( TrackedEntityInstanceCriteria criter
         throws WebMessageException,
```

Comment:
You can remove these two exceptions: no longer thrown

Reply:
done, ty